src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -100,7 +100,7 @@ chip::Controller::CommissioningParameters sCommissioningParameters;
 
 chip::Controller::ScriptDevicePairingDelegate sPairingDelegate;
 chip::Controller::ScriptPairingDeviceDiscoveryDelegate sPairingDeviceDiscoveryDelegate;
-chip::Credentials::GroupDataProviderImpl sGroupDataProvider;
+chip::Credentials::GroupDataProviderImpl sGroupDataProvider{ 5, 8 };
```

Comment:
Please make these values configurable at the top with constant (this looked very magical), and start with 50 groups per fabric and 50 group keys per fabric --> this will be more than enough for a lot of testing and we won't run out.

Reply:
I used the config that were already there and just added the defines to the BUILD.gn file for this library